src/games/strategy/triplea/ai/proAI/util/ProTerritoryValueUtils.java

```diff
@@ -119,9 +119,19 @@ public static double findTerritoryAttackValue(final PlayerID player, final Terri
       if (!territoriesThatCantBeHeld.contains(t) && t.isWater()
           && !data.getMap().getNeighbors(t, Matches.TerritoryIsWater).isEmpty()) {
 
+        // Find nearby capitals and factories
+        Set<Territory> neighborTerritories = new HashSet<>();
+        for (int i = 9; i <= 30; i++) {
```

Comment:
Magic numbers with `9` and `30`. Extract to  either method local variables or class constants. I lack context to know if these would potentially be good settings candidate available for a user to set if they have a more/less powerful computer.